src/cli/command-package.ts

```diff
@@ -26,7 +26,9 @@ export default async function run(options: CliOptions) {
 
   // Ok, we're updating a package in a directory not under our control.
   // We need to be sure it's safe.
-  const {stdout, stderr} = await exec(inDir, 'git', ['status', '-s']);
+  const {stdout, stderr} = await exec(inDir, 'git', ['status', '-s'], {
+    maxBuffer: 1024 * 1024,  // The file list can get pretty long.
```

Comment:
It's a good idea but 1) this command doesn't cover untracked files, so we'd need to run two commands (I tried a bit to find a single command that would cover everything `git status` does, but doesn't find one), and 2) it's a bit awkward with `execFile` to check for non-zero exit code, because it throws, and you need to exclude more serious errors like file-not-found. Could totally do this, just doesn't seem worth the work.

Also, I noticed that we were running `git status` even when `force` was true, and then ignoring the result. So I re-arranged the `if` statement to not run the command at all with `force`, which was `true` in the code path I was hitting (updating the test fixtures).